### ISSUE TITLE
[Bug]: Theme check doesn't detect missing translations for schema

### ISSUE THREAD

**Issue by bmandl** [Shopify/theme-check#730]
### Please confirm that you have:

- [X] Searched [existing issues](.) to see if your issue is a duplicate. (If you’ve found a duplicate issue, feel free to add additional information in a comment on it.)
- [X] Reproduced the issue in [the latest CLI version](https://www.npmjs.com/package/@shopify/cli).

### In which of these areas are you experiencing a problem?

Theme

### Expected behavior

Identifies missing or additional translations in locale files.

### Actual behavior

Missing translations not found and no errors reported.
Actual output:
>279 files inspected, 0 offenses detected, 0 offenses auto-correctable

### Verbose output

```shell
2023-04-03T20:14:57.484Z: Running command theme check
  shopify:theme:check init version: @oclif/core@2.1.4 argv: [ '-c', 'translation', '--verbose' ] +0ms
2023-04-03T20:14:57.511Z:
Running system process:
  · Command: ruby -v
  · Working directory: /home/bmandl/shopify/dawn

2023-04-03T20:14:57.586Z:
Running system process:
  · Command: bundle -v
  · Working directory: /home/bmandl/shopify/dawn

2023-04-03T20:14:57.853Z: Reading the content of file at ../../.nvm/versions/node/v18.12.1/lib/node_modules/@shopify/theme/node_modules/@shopify/cli-kit/assets/cli-ruby/Gemfile...
2023-04-03T20:14:57.861Z:
Running system process:
  · Command: bundle install
  · Working directory: /home/bmandl/.nvm/versions/node/v18.12.1/lib/node_modules/@shopify/theme/node_modules/@shopify/cli-kit/assets/cli-ruby

2023-04-03T20:14:58.416Z:
Running system process:
  · Command: bundle exec ruby /home/bmandl/.nvm/versions/node/v18.12.1/lib/node_modules/@shopify/theme/node_modules/@shopify/cli-kit/assets/cli-ruby/bin/shopify theme check . --category translation --output text
  · Working directory: .

Checking . ...
279 files inspected, 0 offenses detected, 0 offenses auto-correctable
2023-04-03T20:15:01.769Z: Failed to report usage analytics: Couldn't find a a package.json traversing directories from /home/bmandl/shopify/dawn
2023-04-03T20:15:01.770Z: Completed command theme check
```

### Reproduction steps

1. Add some new translation in en.default.schema.json file
2. run `shopify theme check --category translation` in theme folder

### Operating System

wsl2 with Ubuntu 20.04 running on Windows 10

### Shopify CLI version (check your project's `package.json` if you're not sure)

3.44.1

### Shell

wsl2 - Ubuntu 20.04

### Node version (run `node -v` if you're not sure)

v18.12.1

### What language and version are you using in your application?

Ruby 3.0.6p216, Node.js v18.12.1

**Comment by lukeh-shopify** [comment]
👋🏻 Hi @bmandl! Thanks for reporting this. Unfortunately the Ruby version of theme check has been deprecated for some time. Would you be able to confirm you are still having issues when using our [maintained version of theme check](https://shopify.dev/docs/storefronts/themes/tools/theme-check)? This is done using either the latest version of the Shopify CLI, or via the Shopify Liquid Visual Studio Code extension.

If you are still having issues, please raise an issue over in the [CLI](https://github.com/Shopify/cli) or [theme tools](https://github.com/Shopify/theme-tools) repositories. Thanks!
